src/lib/common/clockFunctions.cpp

```diff
@@ -61,3 +61,21 @@ void clock_addtime(struct timespec* accTime, struct timespec* diffTime)
     accTime->tv_nsec -= 1000000000;
   }
 }
+
+
+
+/* ****************************************************************************
+*
+* clock_subtime - 
+*/
+void clock_subtime(struct timespec* subtrahend, struct timespec* minuend)
```

Comment:
It would be great to align the signature of this function with the sylte that will be used at the end for clockFunctions.cpp/h being discussed at https://github.com/telefonicaid/fiware-orion/pull/1501/files#r44779953. After lasts comments, I understand it should be:

```
void clock_subtime(struct timespec* subtrahend, const struct timespec* minuend)
```